### PR TITLE
Okta 461740 a11y issue qrcode ov setup page

### PR DESCRIPTION
**Pull request by praveendurai-okta**
## Description:

Update instructions in OV Enrollment UI to make it clear for screen readers.

Okta Classic OV QR code enrollment page:

Change page description text : “Launch Okta Verify on your mobile device and select “Add an account”. Scan the QR code to continue.“

Change “Can’t scan?” aria description to “Setup without scanning a QR code.“

OIE OV QR code enrollment page:

Change “Can’t scan?” aria description to “Setup without scanning a QR code.“

## PR Checklist

- [X] Have you verified the basic functionality for this change?

- [X] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Screenshot/Video:

https://user-images.githubusercontent.com/94395085/151614054-e32dc35e-e25a-4b48-ad96-6b70ee0aa6d4.mp4

https://user-images.githubusercontent.com/94395085/151614057-d6e49f95-2aca-4207-8823-513a0ae1a6f3.mp4

### Reviewers:
@mauriciocastillosilva-okta  

### Issue:

- [OKTA-461740](https://oktainc.atlassian.net/browse/OKTA-461740)

### CODE REVIEW

File: packages/@okta/i18n/src/properties/login.properties [hunk 1]
```diff
@@ -513,10 +513,11 @@ enroll.totp.enterCode = Enter code displayed from the application
 
 ## Barcode View
 # {0} is the name of a MFA factor
-enroll.totp.setupApp = Launch {0} application on your mobile device and select Add an account.
+enroll.totp.setupApp = Launch {0} on your mobile device and select “Add an account”. Scan the QR code to continue.
```
Comment (indirathirumalaimurugan-okta) [inline packages/@okta/i18n/src/properties/login.properties:516]: can some test be added to verify this text?

Comment (praveendurai-okta) [reply packages/@okta/i18n/src/properties/login.properties:516]: added.

File: packages/@okta/i18n/src/properties/login.properties [hunk 2]
```diff
@@ -513,10 +513,11 @@ enroll.totp.enterCode = Enter code displayed from the application
...
 # {0} is the name of a MFA factor
 enroll.totp.setupGoogleAuthApp = Launch {0}, tap the "+" icon, then select "Scan barcode".
 enroll.totp.cannotScan = Can't scan?
+enroll.totp.aria.cannotScan = Setup without scanning a QR code.
```
Comment (indirathirumalaimurugan-okta) [inline packages/@okta/i18n/src/properties/login.properties:520]: does this require adding a new key to `i18nTransformer.js`? @nikhilvenkatraman-okta

Comment (nikhilvenkatraman-okta) [reply packages/@okta/i18n/src/properties/login.properties:520]: The easiest way to know if we need to add it to i18nTransformer is probably checking the UI by loading it in `ok_PL` language. Given that this is used within the `aria-label` attribute, I am guessing this does not get rendered on the UI.

Comment (praveendurai-okta) [reply packages/@okta/i18n/src/properties/login.properties:520]: Yes it will not be loaded in UI. when i checked in DOM, i could see the encoded value.

Comment (mauriciocastillosilva-okta) [reply packages/@okta/i18n/src/properties/login.properties:520]: I don't think it does, the aria label does not map to any field in the API response.

**Review by mauriciocastillosilva-okta — APPROVED** [review]
looks good

**Review by nikhilvenkatraman-okta — APPROVED** [review]

**Review by indirathirumalaimurugan-okta — APPROVED** [review]
lgtm

**Review by lesterchoi-okta — APPROVED** [review]

**Pull request closed without merging**